sdks/wasm/packager.cs

```diff
@@ -758,15 +785,40 @@ class WasmOptions {
 			ninja.WriteLine ($"build $appdir/mono.js: emcc-link $builddir/driver.o {ofiles} {profiler_libs} {runtime_libs} $mono_sdkdir/wasm-runtime-release/lib/libmono-native.a | $tool_prefix/library_mono.js $tool_prefix/binding_support.js $tool_prefix/dotnet_support.js");
 		}
 		if (enable_linker) {
+
+			switch (linkMode) {
+			case LinkMode.SdkOnly:
+				coremode = "link";
+				usermode = "copy";
+				break;
+			case LinkMode.Full:
+				coremode = "link";
+				usermode = "link";
+				break;
+			default:
+				coremode = "link";
+				usermode = "link";
+				break;
+			}
+
 			string linker_args = "";
 			foreach (var assembly in root_assemblies) {
 				string filename = Path.GetFileName (assembly);
 				linker_args += $"-a linker-in/{filename} ";
 			}
-			foreach (var assembly in dontlink_assemblies.Keys) {
-				linker_args += $"-p copy {assembly} ";
+
+			// the linker does not consider these core by default
+			// add netstandard as a core library
+			linker_args += $"-p {coremode} netstandard ";
```

Comment:
This should be fixed in linker

Reply:
Marek.  We can add it here:  https://github.com/mono/linker/blob/47cd4809b68b1782914e0395aa0f744e80f9f344/linker/Linker/LinkContext.cs#L312

Reply:
https://github.com/mono/linker/pull/448

Reply:
The linker is bumped which includes the fix for `netstandard` as stated above : https://github.com/mono/mono/pull/12560